nevergrad/optimization/recorded_recommendations.csv

```diff
@@ -46,7 +46,7 @@ OnePointDE,0.5018723038,0.1388461772,0.6407611133,0.5273326651,,,,,,,,,,,,
 OptimisticDiscreteOnePlusOne,0.7531428339,0.0,0.0,1.095956118,,,,,,,,,,,,
 OptimisticNoisyOnePlusOne,0.0,0.0,0.0,0.0,,,,,,,,,,,,
 PCEDA,0.0,0.0,0.0,0.0,,,,,,,,,,,,
-PSO,-0.1160475434,-0.6432675407,0.9323470275,-1.0235304297,6.3613408897,6.3613408897,0.6140489502,-6.3613409024,-0.3554047402,-0.4857831932,,,,,,
+PSO,0.5972794706,0.0090103539,-0.1573973735,6.3613408897,0.9015164536,1.0457921428,-0.2443563036,-6.3613409024,-0.2492190243,-0.4584048136,,,,,,
```

Comment:
Change is only due to the queue being initialized in reversed order because of `extend`. Results are otherwise unmodified.